source/ui/REIXS/REIXSSampleChamberButtonPanel.h

```diff
@@ -50,6 +52,7 @@ protected slots:
 
 	QToolButton *createQToolButton(QString text, QIcon icon);
 	AMControlMoveButton *createAMControlMoveButton(QString text, QIcon icon);
+	AMXYThetaControlMoveButton *createAMXYThetaControlMoveButton(QString text, QIcon icon);
```

Comment:
Yeah.  It uses implicit sharing which minimizes the amount of deep-copying that needs to be done.